modules/adfinityBidAdapter.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+```
+Module Name: Adfinity Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: adfinity_prebid@i.ua
+```
+
+# Description
+
+Module that connects to Adfinity demand sources
+
+# Test Parameters
+```
+    var adUnits = [{
+                code: 'placementid_0',
+                mediaTypes: {
+                    banner: {
+                        sizes: [[300, 250], [300,600]]
+                    }
+                },
+                bids: [{
+                        bidder: 'afinity',
```

Comment:
small typo in the bidder name here.  Should be `adfinity` as per the `BIDDER_CODE` in the .js file.

Reply:
I've added test params for video and native into .md file. For testing just change traffic type.
About gzip, is this mandatory in current release?

Reply:
Thanks for updating the test params, I'll take a look.  

The compression of the response is required; this can be done with `gzip` or other similar settings in the `content-encoding` param (such as `deflate` or `compress`, if those are more preferable).